### DISCUSSION TITLE
Any ETA on the Semver feature?

### DISCUSSION THREAD

**Discussion by philippeboyd** [hairyhenderson/gomplate#1924] [General]
Hi @hairyhenderson ,

Do you have any ETA on the release of the [SemVer feature](https://docs.gomplate.ca/functions/semver/)? 
I'm aware that `This function is in development, and not yet available in released builds of gomplate.`

Would it be possible to at least have it released with `--experimental` ?

**Comment by hairyhenderson** [comment]
That'll make it in v4, which is coming _eventually_... I can't make estimates given how sporadic my available time is for gomplate.

The `--experimental` flag is for features which are in a release, but not enabled by default due to various factors (like uncertainty around API).

You could build your own binary in the meantime!

**Reply by philippeboyd** [reply]
Sure thing, I've experienced with the latest changes on `main` and it's working well.

Thanks!
